dashboard/src/actions/repos.test.tsx

```diff
@@ -271,41 +263,19 @@ spec:
 
         expect(AppRepository.create).toHaveBeenCalledWith(
           "my-repo",
-          "my-namespace",
           "http://foo.bar",
-          {},
-          { spec: { containers: [{ env: [{ name: "FOO", value: "BAR" }] }] } },
-        );
-      });
-
-      // Example from https://nealpoole.com/blog/2013/06/code-execution-via-yaml-in-js-yaml-nodejs-module/
-      const unsafeYAMLTemplate =
-        '"toString": !<tag:yaml.org,2002:js/function> "function (){very_evil_thing();}"';
-
-      it("does not call AppRepository create with an unsafe pod template", async () => {
```

Comment:
The frontend no longer parses the yaml. It's now text sent straight to the backend API.